r/R/json.R

```diff
@@ -38,6 +38,8 @@
 read_json_arrow <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
   tab <- JsonTableReader$create(file, ...)$Read()$select(!!enquo(col_select))
 
+  tab$ValidateFull()
```

Comment:
@nealrichardson should we toggle this via an option?

Reply:
I don't have an opinion; sounds like you think we should? Do any of the other readers do validation like this at all? (I'm not aware that they do.)

Reply:
Oh I see, this is added in this patch. IMO the library shouldn't be returning invalid Tables, so calling `Validate` on them seems unnecessary. I wouldn't add this to the reader function here; instead, if someone wanted to validate, they could return the Table themselves (`as_data_frame = FALSE`) and then call `$ValidateFull()` on it themselves.

Reply:
I would remove this, it was only to help me debug